src/components/LocationSearch.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import LocationSearch from "./LocationSearch";
+const toLocation = {};
+export default {
+  title: "Location Search",
+  component: LocationSearch,
+};
+export const Primary = () => (
+  <LocationSearch
+    type="To"
+    initialLocation={toLocation}
+    locationChange={() => {}}
```

Comment:
this be onLocationChange - like DOM event

Reply:
Will you please explain the change? Do we have to have to rename locationChange or change the function in locationChange?

Reply:
usual DOM events are like onClick, onBlur, on<Event>; Here the event is location is changing so we can rename the function as onLocationChange

Reply:
Updated